webapp/src/app/api/messages/route.ts

```diff
@@ -11,18 +12,17 @@ export async function GET() {
     const msgAdapter = MessageAdapterFactory.createAdapter(config);
     const messages = await msgAdapter.getMessages();
 
-    // TODO: change data instruction to be a map of key to value, instead of object
```

Comment:
remove it cuz the current format looks ok, and I can't remember why I add this TODO.